packages/python-packages/apiview-gpt/ci.yml

```diff
@@ -15,3 +29,7 @@ extends:
     FeedName: 'public/azure-sdk-for-python'
     ArtifactName: 'apiviewcopilot'
     PackageName: 'Python API View Copilot'
+    TestSteps:
+    - script: |
+        echo "Pass Check Enforcer!"
+      displayName: 'Pass Check Enforcer'
```

Comment:
@weshaggard @scbedd what is the minimal CI you would recommend in order to get this pipeline to run so Check Enforcer is happy? It's really annoying to have to override it for every PR.

Reply:
Need the build def created that honors it too. I'm executing now give me a couple.